webapp-django/crashstats/symbols/views.py

```diff
@@ -95,6 +95,12 @@ def unpack_and_upload(iterator, symbols_upload, bucket_name, bucket_location):
 
             file = StringIO()
             file.write(member.extractor().read())
+
+            for ext in settings.SYMBOLS_MIME_OVERRIDES:
+                if key_name.endswith(ext):
```

Comment:
Should we use `os.path.splitext()` instead? e.g. if one of the keys in `SYMBOLS_MIME_OVERRIDES` is 'ar' this'd be true if the file is `foo.bar` and `foo.tar`.